packages/material-ui/src/Select/Select.test.js

```diff
@@ -93,27 +93,14 @@ describe('<Select />', () => {
     // this means the trigger is blurred immediately
     const handleBlur = spy();
     const { getByRole, getAllByRole, queryByRole } = render(
-      <Select
```

Comment:
Can we leave this? While it's not necessarily required for the test it simulates how an actual user-click would look like.

Reply:
The select focus on mouse down internally. Do you still want to keep it?

Reply:
Yes because as I said: This is closer to actual user behavior.

Reply:
Ok, I will revert it :)

Reply:
It sounds like we are on a limit of the test infrastructure. It would be interesting if this could be supported directly. I'm wondering if it's something testing-library has an opportunity with.

Reply:
It's a bit tricky since this behavior is platform dependent. I just want to avoid unnecessary test changes to be as sure as possible that we understand the new behavior.